test/integration/controller/jobs/pod/pod_controller_test.go

```diff
@@ -1828,6 +1829,8 @@ var _ = ginkgo.Describe("Pod controller interacting with Workload controller whe
 						Message: "The workload is deactivated due to exceeding the maximum number of re-queuing retries",
 					}, util.IgnoreConditionTimestampsAndObservedGeneration),
 				))
+				// Fake client that are using on unit tests doesn't allow us to check it.
```

Comment:
Same suggestions here.

Reply:
In this case it is enough if the message is changed and the test will pass even if the condition stays.

Reply:
Removed